### ISSUE TITLE
Please provide documentation for twig 

### ISSUE THREAD

**Issue by DoNormal** [zicht/itertools#41]
There is no documentation for itertools in Twig. Sometimes it's easy to guess (`data|map('id')` but more elaborate stuff is missing. E.g. the documentation of `filter` in the readme.md is only applicable to php.

Adding a few code examples would help a lot.

```
data|filter(filtering('equals', item.id, 'page_id'))
```

That would make this library even better ✨

**Comment by boudewijn-zicht** [comment]
Yes, this has been available for a while now.  See the readme file.
